### DISCUSSION TITLE
Does anyone know what happened to Electron (the file based proxy)?

### DISCUSSION THREAD

**Discussion by KandyKaneMarvel** [3kh0/ext-remover#1036] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have genuinely not heard a soul talk about this ever, and I don't remember where it came from. It's deprecated now, but it used to/might still work (my district blocked all repl links so I don't know if it's been taken down or something.

Anyways, here's the file:
[Electron.zip](https://github.com/3kh0/ext-remover/files/15204517/Electron.zip)

If anyone has information about this project or similar ones, this would be the place to post it.

**Comment by myschoolisass** [comment, marked as answer]
![image](https://github.com/3kh0/ext-remover/assets/148739396/1a386ef4-77b6-47fd-98de-0fd4506d6f04)
this proxy uses an outside source as its proxy, so its not file based. this basically is just [bunker v6](https://github.com/Brandon421-ops/Bunker-v6-Proxy-And-Browser) (this is a person who reuploaded it, the original by [https://github.com/JavaScythe](javascythe)) but it still has a proxy on it
i'm not quite sure, but it also looks like it was written with chatgpt (since i use it a lot to skid i would know)
the file itself sends two errors: it's trying to frame replit.com (probably a relic of it being old and the proxy in use on electron still using repl.co) and another which is a issue in the script which i dont know why it doesn't work (i dont want to spend too much time looking thru a deprecated proxy iframer)
the proxy was written by [proud parrot](https://github.com/proudparrot2), so something similar might be [artclass](https://github.com/art-class/v4), a proxy that he worked on (the proxy that is used in electron is actually a fork of [cohen erickson's](https://replit.com/@CohenErickson) [ultraviolet browser](https://replit.com/@CohenErickson/Ultraviolet-Browser%23index.js))

and just to note, i have _never_ heard of this

**Reply by KandyKaneMarvel** [reply]
Thank's homedog. Was wondering where this came from because I've had it on my Chromebook for ~two years.
